src/assets/styles/header.scss

```diff
@@ -66,15 +66,22 @@
   }
 }
 
+@keyframes slideUp {
```

Comment:
I dig the animation, but one thing it changes is the design, where the logos are maintained above the filters:

![image](https://cloud.githubusercontent.com/assets/1724544/24768256/cc4482c6-1ab6-11e7-9112-7dbc236acab3.png)

There is some code that makes that work by calculating the right position and setting an inline style: https://github.com/aclu-people-power/map/blob/master/src/components/Header.js#L35

So I'm not sure. Maybe check with one of the design folks? If they are cool with the full screen style, that's nice cause the animation is sweet, and we could drop that garbagey code setting an inline style too :grin: (Also could wire up the animation to work with the inline style thing but I suspect that might get complicated)